cpp/test/data/new_super_capacitor.info

```diff
@@ -0,0 +1,126 @@
+type New_SuperCapacitor
```

Comment:
is that file any different from the old one beside ``type``?
if the answer is no please user the legacy one and just overwrite ``type``
```
ptree.put('type', 'New_SuperCapacitor')
```